src/base/search/searchpluginmanager.cpp

```diff
@@ -236,9 +242,11 @@ void SearchPluginManager::installPlugin_impl(const QString &name, const QString
             Utils::Fs::forceRemove(destPath + ".bak");
             // Update supported plugins
             update();
+            qDebug() << "Plugin is not supported.";
             emit pluginUpdateFailed(name, tr("Plugin is not supported."));
         }
         else {
+            qDebug() << "Plugin is not supported.";
```

Comment:
can you omit these? 
If there are any messages should print out, it should go through `Logger`.

Reply:
"Plugin is not supported." without any particular information is quite meaningless message.

Reply:
> If there are any messages should print out, it should go through Logger.

`qDebug` and `Logger` are both used pretty extensively in the codebase. Is the `qDebug` way of logging deprecated?

Reply:
>Is the qDebug way of logging deprecated?

IMO yes, we don't need to distribute these code/messages unless we want users to help debug something, even so, we can provide a discrete build for them instead of using qDebug.

Reply:
I'll use the logger so that they'll be visible when the Log is added to the webui. I'll also do this for when plugin updates fail - though this is admittedly a poor solution.

Reply:
IMO, qDebug is still usefull for dev-eyes-only messages (e.g. to indicate some important execution points etc.). Also you free to use it as you want when you test some new code, but many of them are redundant in result and should be removed.